docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -209,9 +209,12 @@ List<String> namesButEmmanuels = persons
     .map(p -> p.name.toLowerCase() )
     .filter( n -> ! "emmanuel".equals(n) )
     .collect(Collectors.toList());
+persons.close(); // close the stream to release I/O resources
```

Comment:
Better use a try with resources IMHO.

Reply:
Yes,  I agree,  was too lazy to refactor the code ... will do it tomorrow